pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/PhysicalPlanVisitor.java

```diff
@@ -56,18 +59,24 @@
  * @see org.apache.pinot.query.runtime.QueryRunner#processQuery(DistributedStagePlan, ExecutorService, Map)
  */
 public class PhysicalPlanVisitor implements StageNodeVisitor<Operator<TransferableBlock>, PlanRequestContext> {
-  private static final PhysicalPlanVisitor INSTANCE = new PhysicalPlanVisitor();
 
-  public static Operator<TransferableBlock> build(StageNode node, PlanRequestContext context) {
-    return node.visit(INSTANCE, context);
+  private List<MailboxIdentifier> _inputMailboxIds = new ArrayList<>();
```

Comment:
can we move this into the PlanRequestContext? otherwise, we need to adjust the comment for the static usage of this visitor class